akka-http-core/src/main/scala/akka/http/scaladsl/model/Uri.scala

```diff
@@ -576,7 +576,7 @@ object Uri {
     }
     def toMultiMap: Map[String, List[String]] = {
       @tailrec def append(map: Map[String, List[String]], q: Query): Map[String, List[String]] =
-        if (q.isEmpty) map else append(map.updated(q.key, q.value :: map.getOrElse(q.key, Nil)), q.tail)
+        if (q.isEmpty) map else append(map.updated(q.key, map.getOrElse(q.key, Nil) :+ q.value), q.tail)
```

Comment:
I agree it's not the most conventional way, but it is possible to submit an arbitrarily big number of parameters with the same key.
Let's see what the others think about it :)

Thanks for the PR, BTW!